pyInterface/module/pyPartialWaveFit/_startParameterGenerator.py

```diff
@@ -47,6 +48,13 @@ def __call__(self):
 		# this sets the amplitues of real-valued waves (e.g. reference wave) to real
 		paraNegLlhd = [amplVectors, np.array([1]*self.model.nmbDatasets)] + [1]*self.model.parameterMapping.nmbAdditionalParameters
 		paraFitter = self.model.parameterMapping.paraLlhd2Fitter(self.model.parameterMapping.paraNegLlhd2Llhd(paraNegLlhd))
+
+		# change real waves to real AND positive
+		if self.constrainPosReal and hasattr(self.model.parameterMapping, "indicesImagFitterPara"):
+			for i,idxIFP in enumerate(self.model.parameterMapping.indicesImagFitterPara):
+				if idxIFP is None and self.model.parameterMapping.indicesRealFitterPara[i] is not None:
+					paraFitter[self.model.parameterMapping.indicesRealFitterPara[i]] = abs(paraFitter[self.model.parameterMapping.indicesRealFitterPara[i]])
+
```

Comment:
Why to you use the `indicesImagFitterPara` here?
This attribute is only implemented for the `ParameterMappingConnected`.
So setting the `constrainPosReal` for the not-connected likelihood would not work.
If you set the constrain for the fitter, which you do by default, this leads to invalid start parameters for the not-connected likelihood, as far is I can see.